### PR TITLE
fix actions so that upstream connects can pass the normal response

### PR DESCRIPTION
**Pull request by coreypurcell**
the upstream connect is going to post async to the downstream connect
and will pass along it's normal conduit response not a serialized line.
The serialized line is only there during submission not completion. We
don't actually need the serialized line in the response. Add the ability
to return a mocked response for testing downstream systems.

### CODE REVIEW

File: spec/conduit/atom/actions/deactivate_spec.rb [hunk 1]
```diff
@@ -58,27 +58,7 @@
 
   context 'when submitting the request' do
     let(:serializable_hash) do
-      {
```
Comment (pedromartinez) [inline spec/conduit/atom/actions/deactivate_spec.rb]: Should these be replaced with parser tests?

Comment (coreypurcell) [inline spec/conduit/atom/actions/deactivate_spec.rb]: the parsers are very bare now. I can move them to their own file or just remove them

Comment (pedromartinez) [inline spec/conduit/atom/actions/deactivate_spec.rb]: They still do pass back information though, right? And we have no test that proves they are "parsing" correctly.

I could just not understand what you mean, though.

Comment (coreypurcell) [inline spec/conduit/atom/actions/deactivate_spec.rb]: they do not pass information back. just the result status. Connect does not utilize any of the old line_attributes, it just verifies the it got a successful response

Comment (coreypurcell) [inline spec/conduit/atom/actions/deactivate_spec.rb]: I believe conduit-sprint is the same, it just tells you if it succeeded

Comment (pedromartinez) [inline spec/conduit/atom/actions/deactivate_spec.rb]: Oh okay. I understand now. I thought this was a global change to all actions.

**Pull request merged**